web/src/App.vue

```diff
@@ -10,16 +18,39 @@
 
 <script>
 import Toolbar from './components/Toolbar.vue';
+import Sidebar from './components/Sidebar.vue';
 
 export default {
   name: 'App',
 
   components: {
     Toolbar,
+    Sidebar,
   },
-
   data: () => ({
-    //
+    showSidebar: false,
   }),
+  computed: {
```

Comment:
FYI I made this a computed property because once we add in i18n, we will need to dynamically infer the titles.